examples/interactive_remote_hyperdrive_example.py

```diff
@@ -33,63 +25,142 @@
 
 # Need to set different db port here to avoid port collisions with local chain
 chain = Chain(rpc_uri, config=Chain.Config(db_port=1234))
-hyperdrive_config = Hyperdrive.Config()
-hyperdrive_pool = Hyperdrive(chain, hyperdrive_address, hyperdrive_config)
 
 # %%
 
+# Initialize agents
+
 # We set the private key here. In practice, this would be in a private
 # env file somewhere, and we only access this through environment variables.
 # For now, we generate a random key and explicitly fund it
-private_key = make_private_key()
-
-# Init from private key and attach policy
-# This ties the hyperdrive_agent to the hyperdrive_pool here.
-# We can connect to another hyperdrive pool and create a separate
-# agent object using the same private key, but the underlying wallet
-# object would then be out of date if both agents are making trades.
-# TODO add registry of public key to the chain object, preventing this from happening
-agent0 = hyperdrive_pool.init_agent(
-    private_key=private_key,
+private_key_0 = make_private_key()
+private_key_1 = make_private_key()
+
+# Init from private key
+agent0 = chain.init_agent(
+    private_key=private_key_0,
+    name="agent0",
+)
+# We can initialize an agent with a custom policy - more on that later
```

Comment:
No custom policy discussion in this file like the local example. Probably makes sense to point them to the other example.

Reply:
Again, I meant active policy here